tests/gcp/hooks/test_dataflow.py

```diff
@@ -346,6 +348,15 @@ def test_start_template_dataflow_with_runtime_env(self, mock_conn, mock_dataflow
             gcsPath=TEMPLATE,
             body=body,
         )
+        mock_dataflowjob.assert_called_once_with(
+            dataflow=mock_conn.return_value,
+            job_id=TEST_JOB_ID,
+            location='us-central1',
+            name='test-dataflow-pipeline-{}'.format(MOCK_UUID),
+            num_retries=5,
+            poll_sleep=10,
+            project_number='test'
+        )
```

Comment:
```suggestion
        )
        mock_uuid.assert_called_once_with()
```

I think it is always better to check that the patched object at least has been called instead of pylint-ignoring it. So that if the underlying uuid generation in the class to test changes somehow it will raise an error.

Reply:
This mock is being checked because the result of calling this function appears in tests, but I've added assertions. Updated. Thanks.